### DISCUSSION TITLE
草了

### DISCUSSION THREAD

**Discussion by ypsywww** [share121/inter-knot#141] [其他]
刚才打了一大段加速器掉了😥全没了
随便打点玩玩吧
![e4dde71190ef76c6a7efaa289344eafaaf51f3dec92c](https://github.com/user-attachments/assets/34943f8c-db07-4691-a1b9-2a9a291bbf9f)

**Comment by AL-Pinecore** [comment]
intelliJ，爱了

**Reply by ypsywww** [reply]
我自己已经找不到我这帖子了
